source/reference/mongosqld.txt

```diff
@@ -29,6 +29,9 @@ Description
 proxies those requests to a :program:`mongod` or :program:`mongos`
 instance.
 
+:program:`mongosqld` enables the :dbcommand:`aggregate`
```

Comment:
There's not a perfect place for this, but I think it belongs much more in https://docs.mongodb.com/bi-connector/master/faq/#how-are-queries-processed, since that's where we mention mongosqld's use of aggregation.